runtime-api/did/src/lib.rs

```diff
@@ -67,11 +67,11 @@ pub type RawDidLinkedInfo<DidIdentifier, AccountId, LinkableAccountId, Balance,
 
 sp_api::decl_runtime_apis! {
 	#[api_version(2)]
-	pub trait Did<DidIdentifier, AccountId, LinkableAccountId, Balance, Key: Ord, BlockNumber> where
+	pub trait Did<DidIdentifier, AccountId, LinkableAccountId, Balance, Key: Ord, BlockNumber: MaxEncodedLen> where
```

Comment:
Why this change?

Reply:
Can you move the trait bound to the `where` clause, so they are grouped together?

Reply:
I get the error when I put the `MaxEncodedLen` in the where block. Looks to me that the macro is broken now.